pkg/machine/ignition.go

```diff
@@ -101,6 +121,15 @@ RequiredBy=multi-user.target
 				Name:     "ready.service",
 				Contents: strToPtr(fmt.Sprintf(ready, "vport1p1", "vport1p1")),
 			},
+			{
+				Name: "docker.service",
+				Mask: boolToPtr(true),
+			},
```

Comment:
I would also mask `docker.socket` and maybe we should set `Enabled: boolToPtr(false)` in these too.

Reply:
Will do